From 188e7c217a942309277c3171de2b863f27453f9a Mon Sep 17 00:00:00 2001
From: nalla <nalla@hamal.uberspace.de>
Date: Tue, 14 Apr 2015 11:42:10 +0100
Subject: [PATCH] man-inline-macro: enable `:linkgit` syntax

For building the documentation with `asciidoctor` the provided
`man-inline-macro` has to be patched to include the `:linkgit` syntax.

Signed-off-by: nalla <nalla@hamal.uberspace.de>
---
 lib/man-inline-macro.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/man-inline-macro.rb b/lib/man-inline-macro.rb
index 08549e9..96718be 100644
--- a/lib/man-inline-macro.rb
+++ b/lib/man-inline-macro.rb
@@ -1,7 +1,7 @@
 RUBY_ENGINE == 'opal' ? (require 'man-inline-macro/extension') : (require_relative 'man-inline-macro/extension')
 
 Extensions.register :uri_schemes do
-  inline_macro ManInlineMacro
+  #inline_macro ManInlineMacro
   # Use the following instead for the git man pages
-  #inline_macro ManInlineMacro, :linkgit
+  inline_macro ManInlineMacro, :linkgit
 end
-- 
2.3.5.windows.8

